.github/workflows/board-automation.yaml

```
@@ -0,0 +1,17 @@
+name: Stripey
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+jobs:
+  add_to_project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: skeet70/default-project-board-action@b1f4d447074827722eb1f0258c3741dd8e156b66
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          github_token: ${{ github.token }}
```
We shouldn't need to create a new token for use with the action, when the action runs with its own temporary token anyway.

---

Fixed!

---

It doesn't look like the fix was pushed, though…

---

thanks for catching...
setting up vscode auto-save :P